### DISCUSSION TITLE
Welcome to dnd5e-spellpoints Discussions!

### DISCUSSION THREAD

**Discussion by misthero** [misthero/dnd5e-spellpoints#60] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GMToolbox** [comment]
I started looking at the DMG spell point system and I didn't like how it limits to just once a day for 6th level and higher but I also don't want 10 9th level spells a day happening. The Advanced Spell point system is better but still suffers from high level casters getting fewer spells instead of more without being too OP. A solution I came up with though not sure how it would work in Foundry, is doubling the cost to cast spells of a level each additional time they are cast. This is  using the DMG version, don't think it will work with the Advanced version. For example  a 20th level caster, casts a 9th level spell for 13 points. The next 9th level spell cast would cost 26. The next one would cost 39. The next would be 52. At this point after casting 4 9th level spells they would have used up 130 spell points. So they do get to cast more than 1 9th which is the whole point of the system but it scales a little better so they don't go too nova. 
It doesn't have to be all spells of the same level either. For instance cast a 6th level spell for 9 points, then a 7th level spell would end up costing 20 since its your second spell of the day over level 6 it doubles its base cost. Then you cast an 8th level for 33 points (3rd spell of the day over 6th costing triple its base cost). And so on. This will limit full casters to only 4 spells a day over level 6 but they can cast what they want at the cost of more spell points spent.

Thoughts on this? Do you think this would be something worth adding?

**Comment by misthero** [comment]
> So they do get to cast more than 1 9th which is the whole point of the system but it scales a little better so they don't go too nova.

Hello @GMToolbox,

I don't know why you say this. The spell point system is a way to give flexibilty not more high level spells, casting 2 level 9 spells is already a big OP boost. That's why the official variant rule artificially limit the number of spells you can cast at higher level to 1. Usually a single level 9 spell is enough to end every fight. Unless you wanna destroy every adventure around there.

Also in my experience playing around level 20 is something you seldom do, most games, published adventures and campaigns all goes between level 3-14. I think you should check your math around those levels unless you are looking for something just for a single level 20 adventure.

What I think is that what you imagine have a lot of complexity and number tracking, and math that will slow down your game if you play live, but since we are making foundry do the calculations for us...

You can enter formulas instead of static numbers in the cost calculation box per slot level using the spell points module. Maybe you can use a macro to update the number of spells already casted in a resource box and multiply that value for the current slot cost.

![immagine](https://github.com/misthero/dnd5e-spellpoints/assets/3662610/13b5d845-0502-4913-8f94-154ba894702b)

![immagine](https://github.com/misthero/dnd5e-spellpoints/assets/3662610/42ac7da4-2aaa-40b4-aec9-f2d1305c4bbb)

The resource should start with 1. The first spell you cast will cost 1 times the listed cost.
At this point you should have a macro that update the resource adding 1.
Next time you cast any spell it will cost double, than 3 times and so on...

I don't have the macro anyway, for that you could try asking on the official foundry discord in the macro channel.

good luck! :)
